.github/ISSUE_TEMPLATE/us-rgbz-mapping.md

```diff
@@ -0,0 +1,21 @@
+---
+name:  RGBZ-mapping US xx
+about: RGBZ veld mapping bij user story
+---
+
+Mapping van velden behorend bij user story #xx
+
+
+### Request
+| RGBZ mapping | Veld in huidige app | Verklaring |
+| --- | --- | --- |
+| `ZAAK/ZaakIdentificatie` | `zaakid` | Uniek volgnummer zaak |
+| ` ` | ` ` | -   |
+| ` ` | ` ` | -   |
+
+### Answer
+| RGBZ mapping | Veld in huidige app | Verklaring |
+| --- | --- | --- |
+| `ZAAK/Registratiedatum` | `zaakdatum` | Registratiedatum Zaak |
```

Comment:
De omschrijving vanuit het RGBZ is "De datum waarop de zaakbehandelende organisatie de ZAAK 
heeft geregistreerd"